### ISSUE TITLE
Default example of MNIST_pytorch shows no output data

### ISSUE THREAD

**Issue by CSH360** [microsoft/nni#3147]
**Environment**: 
- NNI version: 1.9
- NNI mode (local|remote|pai): local mode
- Client OS: Windows10
- Python version: 3.7.9
- PyTorch version: 1.7.0
- Is conda/virtualenv/venv used?: conda
- Is running in Docker?: No

**Log message**:
 - nnimanager.log:
[2020-12-2 1:56:33 PM] INFO [ 'Datastore initialization done' ]
[2020-12-2 1:56:33 PM] INFO [ 'RestServer start' ]
[2020-12-2 1:56:33 PM] INFO [ 'Construct local machine training service.' ]
[2020-12-2 1:56:33 PM] INFO [ 'RestServer base port is 8080' ]
[2020-12-2 1:56:33 PM] INFO [ 'Rest server listening on: http://0.0.0.0:8080' ]
[2020-12-2 1:56:36 PM] INFO [ 'NNIManager setClusterMetadata, key: trial_config, value: {"command":"python3 mnist.py","codeDir":"C:\\\\Users\\\\User\\\\nni\\\\examples\\\\trials\\\\mnist-pytorch\\\\.","gpuNum":0}' ]
[2020-12-2 1:56:36 PM] INFO [ 'required GPU number is 0' ]
[2020-12-2 1:56:38 PM] INFO [ 'Starting experiment: sLBmCBch' ]
[2020-12-2 1:56:38 PM] INFO [ 'Change NNIManager status from: INITIALIZED to: RUNNING' ]
[2020-12-2 1:56:38 PM] INFO [ 'Add event listeners' ]
[2020-12-2 1:56:38 PM] INFO [ 'Run local machine training service.' ]
[2020-12-2 1:56:38 PM] INFO [ 'NNIManager received command from dispatcher: ID, ' ]
[2020-12-2 1:56:38 PM] INFO [ 'NNIManager received command from dispatcher: TR, {"parameter_id": 0, "parameter_source": "algorithm", "parameters": {"batch_size": 128, "hidden_size": 128, "lr": 0.1, "momentum": 0.7204432483523889}, "parameter_index": 0}' ]
[2020-12-2 1:56:43 PM] INFO [ 'submitTrialJob: form: {"sequenceId":0,"hyperParameters":{"value":"{\\"parameter_id\\": 0, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"batch_size\\": 128, \\"hidden_size\\": 128, \\"lr\\": 0.1, \\"momentum\\": 0.7204432483523889}, \\"parameter_index\\": 0}","index":0}}' ]
[2020-12-2 1:56:53 PM] INFO [ 'Trial job wqG35 status changed from WAITING to FAILED' ]
[2020-12-2 1:56:53 PM] INFO [ 'NNIManager received command from dispatcher: TR, {"parameter_id": 1, "parameter_source": "algorithm", "parameters": {"batch_size": 32, "hidden_size": 512, "lr": 0.0001, "momentum": 0.9362915267383966}, "parameter_index": 0}' ]
[2020-12-2 1:56:58 PM] INFO [ 'submitTrialJob: form: {"sequenceId":1,"hyperParameters":{"value":"{\\"parameter_id\\": 1, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"batch_size\\": 32, \\"hidden_size\\": 512, \\"lr\\": 0.0001, \\"momentum\\": 0.9362915267383966}, \\"parameter_index\\": 0}","index":0}}' ]
[2020-12-2 1:57:03 PM] INFO [ 'Trial job VSd3x status changed from WAITING to FAILED' ]
[2020-12-2 1:57:03 PM] INFO [ 'NNIManager received command from dispatcher: TR, {"parameter_id": 2, "parameter_source": "algorithm", "parameters": {"batch_size": 16, "hidden_size": 128, "lr": 0.1, "momentum": 0.9768506810158324}, "parameter_index": 0}' ]
[2020-12-2 1:57:08 PM] INFO [ 'submitTrialJob: form: {"sequenceId":2,"hyperParameters":{"value":"{\\"parameter_id\\": 2, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"batch_size\\": 16, \\"hidden_size\\": 128, \\"lr\\": 0.1, \\"momentum\\": 0.9768506810158324}, \\"parameter_index\\": 0}","index":0}}' ]
[2020-12-2 1:57:14 PM] INFO [ 'Trial job pzlbW status changed from WAITING to FAILED' ]
[2020-12-2 1:57:14 PM] INFO [ 'NNIManager received command from dispatcher: TR, {"parameter_id": 3, "parameter_source": "algorithm", "parameters": {"batch_size": 32, "hidden_size": 512, "lr": 0.1, "momentum": 0.3089412555884429}, "parameter_index": 0}' ]
 
 - dispatcher.log:
[12/02/2020, 01:56:38 PM] INFO (nni.msg_dispatcher_base/MainThread) Start dispatcher
[12/02/2020, 01:56:38 PM] INFO (hyperopt.tpe/Thread-1) tpe_transform took 0.015627 seconds
[12/02/2020, 01:56:38 PM] INFO (hyperopt.tpe/Thread-1) TPE using 0 trials
[12/02/2020, 01:56:53 PM] INFO (hyperopt.tpe/Thread-1) tpe_transform took 0.000000 seconds
[12/02/2020, 01:56:53 PM] INFO (hyperopt.tpe/Thread-1) TPE using 0 trials

 - nnictl stdout and stderr:
Id: sLBmCBch    Name: example_mnist_pytorch    Status: DONE    Port: 8080    Platform: local    StartTime: 2020/12/02 13:56:38    EndTime: 2020/12/02 13:58:31
Id: sLBmCBch    Name: example_mnist_pytorch    Status: DONE    Port: 8080    Platform: local    StartTime: 2020/12/02 13:56:38    EndTime: 2020/12/02 13:58:31

**What issue meet, what's expected?**:
I use the command to run NNI's default example of MNIST-pytorch as below

nnictl create --config nni\examples\trials\mnist-pytorch\config.yml

However, metric on WebUI is empty. Has anyone bumped into the same problem before and how to fix it? THX a lot!

![未命名](https://user-images.githubusercontent.com/72777983/100834990-d739bc00-34a7-11eb-8111-ceb4174d9ad7.jpg)

**Comment by liuzhe-lz** [comment]
Please check `C:\Users\<username>\nni-experiments\<experiment-id>\trials\<trial-id>\stderrr`. Maybe you need to change `python3` in config file to `python`.

**Comment by scarlett2018** [comment]
> Please check `C:\Users\<username>\nni-experiments\<experiment-id>\trials\<trial-id>\stderrr`. Maybe you need to change `python3` in config file to `python`.

@CSH360 - any updates? thanks.

**Comment by kvartet** [comment]
hello @CSH360, I will close this issue as it has not been updated for more than three months, feel free to reopen it if you still think it an active issue.
